src/sprout/Controllers/Admin/PageAdminController.php

```diff
@@ -403,7 +404,7 @@ public function _addSave(&$page_id)
         Pdb::insert('page_revisions', $update_fields);
 
         // Admin permissions
-        if ($_POST['admin_perm_specific'] == 1 and @count($_POST['admin_permissions'])) {
```

Comment:
:scream: I thought we already cleaned these out!

Reply:
Yeah that was a surprise!